extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss

```diff
@@ -101,7 +101,7 @@ $start-a-wikia-logo-size: 30px;
 		}
 	}
 
-	@media #{$breakpoint-small-only} {
+	@media #{$breakpoint-below-small-plus} {
```

Comment:
On the other hand - when we change from breakpoints to any other layout we would need to grep through all extensions to look for this media queries. Having all code related with breakpoints inside breakpoints makes sense. Maybe instead of putting it inside breakpoints-layout.scss we can create file like breakpoints-globalnavigation.scss inside global nav extension. Thanks to this when changing code related to breakpoints we know we need to look inside breakpoints-* files not foo-*.scss